index.js

```diff
@@ -179,13 +225,13 @@ module.exports = (cmd, args, opts) => {
 
 		spawned.on('error', err => {
 			cleanupTimeout();
-			resolve({err});
+			resolve({error: err});
```

Comment:
Renamed `err` to `error` because of the `makeError` function which uses `error`. The reason is that `spawnSync` uses `error` and I thought it would be easier to rename it here. 

I can also do `const err = result.err || result.error` in the `makeError` function. Don't really have a strong opinion on this one.

Reply:
> Renamed err to error because of the makeError function which uses error. The reason is that spawnSync uses error and I thought it would be easier to rename it here.

👍